src/auth/auth.controller.ts

```diff
@@ -37,4 +39,15 @@ export class AuthController {
   public async signUpWithEmailAndPassword(@Body() createUserDto: CreateUserDto) {
     return this.authService.signUpWithEmailAndPassword(createUserDto);
   }
+
+  @Get("google")
+  @UseGuards(AuthGuard("google"))
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  async googleAuth(@Req() _req: any) {}
+
+  @Get("google/redirect")
+  @UseGuards(AuthGuard("google"))
+  googleAuthRedirect(@Req() req: any) {
```

Comment:
type `req` to use the express `Request` object

`import { Request } from "express";`

```suggestion
  googleAuthRedirect(@Req() req: Request) {
```

Reply:
Will do